lib/utils/ProductFields.dart

```diff
@@ -101,15 +108,20 @@ extension ProductFieldExtension on ProductField {
     ProductField.CATEGORIES: 'categories',
     ProductField.CATEGORIES_TAGS: 'categories_tags',
     ProductField.CATEGORIES_TAGS_IN_LANGUAGES: 'categories_tags_',
+    ProductField.LABELS: 'labels',
```

Comment:
Probably would be worth checking in a test that when we ask for those fields in a get product query, we actually get them.

Reply:
On it